java/build.gradle

```diff
@@ -17,6 +17,13 @@ subprojects {
     // minimal java compatibility level
     plugins.withId('java') {
         sourceCompatibility = targetCompatibility = "11"
+        dependencies {
```

Comment:
Do we want to add this dependency to all modules?

Reply:
Lets keep this PR only for version update